docs/cep/getting-started-stream-workers.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 10
+title: Getting Started with Stream Workers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Steps from '../_partials/_get-started-steps.md';
+
+This article is an introduction to using stream workers with the Macrometa GDN console.
+
+<Prerequisites />
+
+## Get Started with Stream Workers
+
+This page guides you through creating a stream worker and updating it.
+
+<Steps />
```

Comment:
These are the wrong steps. This is the steps for running code in the IDE. You need to write steps that will get the user to log in and get to the write place in the console to do it. 

The original instructions, which you can view in Prod, are actually not terrible. I suggest using those as a starting point. 

Remember, we want the shortest possible "time to wow" that we can get. The biggest issue I have with the original, aside from the small detail of not telling me how or where to log in, is that it does not explain what I am doing or why very well. I have no clue what "SweetProductionData" and the streams are doing for me.